metal-stack-support.md

```diff
@@ -0,0 +1,83 @@
+# Support for cilium at metal-stack
+
+## Version
+
+Must be based on v1.14.0 because we are still not at g/g 1.50+
+
+## Configuration which worked
+
+``ỳaml
+kind: ConfigMap
+  name: cilium-config
+  namespace: kube-system
+apiVersion: v1
+data:
+  agent-health-port: "9879"
+  agent-not-ready-taint-key: node.cilium.io/agent-not-ready
+  annotate-k8s-node: "true"
+  arping-refresh-period: 30s
+  auto-direct-node-routes: "false"
+  bpf-ct-global-any-max: "262144"
+  bpf-ct-global-tcp-max: "524288"
+  bpf-lb-external-clusterip: "false"
+  bpf-lb-map-max: "65536"
+  bpf-map-dynamic-size-ratio: "0.0025"
+  bpf-nat-global-max: "524288"
+  bpf-policy-map-max: "16384"
+  cgroup-root: /run/cilium/cgroupv2
+  cilium-endpoint-gc-interval: 5m0s
+  cluster-name: default
+  cluster-pool-ipv4-cidr: 10.244.0.0/18
+  cluster-pool-ipv4-mask-size: "24"
+  debug: "false"
+  disable-cnp-status-updates: "true"
```

Comment:
I guess we need to specify `devices: "lan+"` because those are the devices the BPF programs are attached to (for example for NodePort) - currently it is initialized with `devices: "lo"` s. https://docs.cilium.io/en/v1.12/cmdref/cilium-agent/:
```
--devices List of devices facing cluster/external network (used for BPF NodePort, BPF masquerading and host firewall); supports '+' as wildcard in device name, e.g. 'eth+'
```

Reply:
This really helped! It now seems to work properly.